src/rules/property-no-unknown/README.md

```diff
@@ -12,35 +12,37 @@ This rule considers properties defined in the [CSS Specifications and browser sp
 
 This rule ignores variables (`$sass`, `@less`, `--custom-property`).
 
+This rule ignores vendor prefixed properties (`-moz-align-self`, `-webkit-align-self`) .
+
 ## Options
 
 ### `true`
 
-The following patterns are considered warnings:
+The following pattern is considered warnings:
```

Comment:
> Instead of changing from plural to singular let's add another example

Defo, as this is a special trigger phase (used on the website to colour code the examples) and shouldn't be changed.